settings.gradle

```diff
@@ -3,14 +3,14 @@ rootProject.name = 'nva-publication-api'
 include 'publication-constants'
 include 'publication-testing'
 include 'publication-commons'
-include 'doi-requests'
-include 'messages'
-include 'publication-rest'
-include 's3-import-commons'
-include 'publication-doi-commons'
-include 'expansion'
-include 'publication-event-handlers'
-include 'cristin-import'
-include 'resource-enricher'
-include 'datacite-commons'
-include 'publishing-request'
+//include 'doi-requests'
```

Comment:
in the next PRs we will include some modules again, in order to keep the PRs relatively small